src/pages/login/index.js

```diff
@@ -0,0 +1,101 @@
+
+    const [values, setValues] = useState({
+        email: "",
+        password: "",
+    });
+
+    const [error, setError] = useState(undefined);
+    const handleSubmit = async (e) => {
+        e.preventDefault();
+
+        try {
+            const response = await axios.post("/login", values);
+            console.log(response.data);
+            // handle success.
+        } catch (error) {
+            console.error(error);
+            // handle error (example).
+            setError("An error occurred, try again...");
+        }
+        
+
+    }
+    
+    
+    return (
+        <div className={styles.loginPage}>
+
+            <div className={styles.asideContainer}>
+            <CustomizableAside
+                    src={"/images/mainAvatar.png"}
+                    width={110}
+                    height={110}
+                    alt={"avatar"}
+                    asideCustomClass={styles.asideContain}
+                    headerText={"Lorem ipsum lorem ipsum"}
+                    paragraphText={"Sed ut perspiciatis  Nemo enim ipsam voluptatem  fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt. Neque porro quisquam "}
+/>
+            </div>
+
+            <div className={styles.loginContainer}>
+                    <div className={styles.loginImage} >
+                        <Image src="/images/mainAvatar.png" width="70" height="70" priority  alt='' />
+                    </div>
+                    <form onSubmit={handleSubmit} className={styles.form}>
+                        <div className={styles.field}>
+                            <label htmlFor="email" className={styles.formLabel}>Email Address</label>
+                            <input
+                            className={styles.input}
+                            onChange={(e) => setValues({ ...values, email: e.target.value })}
+                            value={values.email}
+                            type="text"
+                            name="email"
+                            placeholder="joe.deo@gmail.com"
+                            required
+                            />
+                        </div>
+                        <div className= {styles.field}>
+                            <label htmlFor="password" className={styles.formLabel}>Password</label>
+                            <input
+                            className={styles.input}
+                            onChange={(e) => setValues({ ...values, password: e.target.value })}
+                            value={values.password}
+                            type="password"
+                            name="password"
+                            placeholder="*********"
+                            required
+                            />
+                            <p className={styles.que}>Forgot Password?</p>
+                        </div>
+                        {error ? <p className= {styles.error}>{error}</p> : <></>}
```

Comment:
@jjleigh For this error code, is it possible to structure it so that we are not rendering an empty fragment (`<></>`) for the unhappy path?

Reply:
We don't need the else case right so I would just do an if condition without the else and eliminate the ternary.
You can use this operator `&&`

```suggestion
              {error && <p className={styles.error}>{error}</p>}
```